src/env-inl.h

```diff
@@ -374,6 +376,10 @@ inline Environment::~Environment() {
   delete[] http_parser_buffer_;
 }
 
+inline NodeOption* Environment::state() const {
```

Comment:
Could you call the class `NodeOptions`, since it encapsulates a whole set of options and not a single one? And ideally rename this method to something like `options()`?

Reply:
Yep, no problem